modular_skyrat/modules/novaya_ert/code/advanced_choice_beacon.dm

```diff
@@ -178,7 +178,7 @@
 /obj/structure/closet/crate/secure/weapon/nri/engineer/offense
 	name = "offensive engineering supplies"
 	loadout_desc = "An assortment of engineering supplies finely tuned for rapid approach defortification and area suppression.\
-	 Features way less barricades and building materials than its more defensive analogue, but includes NRI-issued viscerator grenades and a combat RCD."
+		Features way less barricades and building materials than its more defensive analogue, but includes NRI-issued viscerator grenades and a combat RCD."
```

Comment:
Same goes for this and other multiline strings

Reply:
Fixed.